kitsune/questions/models.py

```diff
@@ -869,7 +868,6 @@ def __str__(self) -> str:
 class AAQConfig(ModelBase):
     title = models.CharField(max_length=255, default="")
     product = models.ForeignKey(Product, on_delete=models.CASCADE, related_name="aaq_configs")
-    pinned_articles = models.ManyToManyField(Document, null=True, blank=True)
```

Comment:
Don't we need to migrate this to the new configuration since we are deleting the field and we already have entries in prod?